src/set_theory/ordinal/arithmetic.lean

```diff
@@ -591,12 +592,31 @@ instance : monoid ordinal.{u} :=
     ⟨⟨prod_punit _, λ a b, by rcases a with ⟨a, ⟨⟨⟩⟩⟩; rcases b with ⟨b, ⟨⟨⟩⟩⟩;
     simp only [prod.lex_def, empty_relation, and_false, or_false]; refl⟩⟩ }
 
-@[simp] theorem type_prod_lex {α β : Type u} (r : α → α → Prop) (s : β → β → Prop)
-  [is_well_order α r] [is_well_order β s] : type (prod.lex s r) = type r * type s := rfl
+theorem mul_def {α β : Type u} (r : α → α → Prop) (s : β → β → Prop)
+  [is_well_order α r] [is_well_order β s] : type r * type s = type (prod.lex s r) := rfl
+
+theorem mul_def_lt {α β : Type u} [has_lt α] [has_lt β]
+  [is_well_order α (<)] [is_well_order β (<)] :
+  @type α (<) _ * @type β (<) _ = @type (β ×ₗ α) (<) _ :=
+rfl
```

Comment:
Given this will appear in the docs with no mention of `β ×ₗ α`, can you mention `lex (β × α)` in a docstring, and similarly below?

Reply:
I don't get it, what should the docstring say?

Reply:
It should say whatever is needed such that a user reading the docs knows what is meant by `type (<) * type (<) = type (<)`; because that's all the context the web docs and autocomplete will show them.

Reply:
So perhaps
```
/-- Note the `(<)` on the RHS is the comparison on `lex (β × α)` via `prod.lex.has_le` -/
```
or whatever the instance is actually called.